src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+const docPopulationArray = document.querySelectorAll('.population');
+
+const docTotal = document.querySelector('.total-population');
+
+const docAverage = document.querySelector('.average-population');
+
+const populationArray = [];
+
+docPopulationArray.forEach((item) => populationArray.push(item.textContent));
+
+const populationNumbers = populationArray.map(
+  (item) => +item.replaceAll(',', ''),
+);
```

Comment:
The conversion from string to number using the unary plus (+) is clever, but for better readability and to avoid confusion, consider using 'parseInt' or 'parseFloat' after replacing the commas.